src/plone/app/mosaic/setuphandlers.py

```diff
@@ -26,6 +28,23 @@
 """
 
 
+@implementer(INonInstallable)
+class HiddenProfiles(object):
+
+    def getNonInstallableProfiles(self):
+        """Hide uninstall profile from site-creation and quickinstaller"""
```

Comment:
docstrings must en with a `.`.

Reply:
we may want to change it in https://github.com/plone/bobtemplates.plone/blob/master/bobtemplates/plone/addon/src/%2Bpackage.namespace%2B/%2Bpackage.name%2B/setuphandlers.py.bob#L10 where i copied it from ;)